php/php.editor/src/org/netbeans/modules/php/editor/actions/FixDuplicateImportStmts.java

```diff
@@ -121,12 +117,10 @@ public void focusLost(FocusEvent arg0) {
         // load localized text into widgets:
         lblTitle.setText(getBundleString("FixDupImportStmts_IntroLbl")); //NOI18N
         lblHeader.setText(getBundleString("FixDupImportStmts_Header")); //NOI18N
-
-        checkUnusedImports = new JCheckBox();
-        Mnemonics.setLocalizedText(checkUnusedImports, getBundleString("FixDupImportStmts_UnusedImports")); //NOI18N
-        bottomPanel.add(checkUnusedImports, BorderLayout.WEST);
-        checkUnusedImports.setEnabled(true);
-        checkUnusedImports.setSelected(removeUnusedImports);
+        fixImportsBottomPanel = new FixImportsBottomPanel(combos, importData);
```

Comment:
I've moved the checkbox and buttons of the bottom panel to the new panel to fix them more easily.

Reply:
Makes sense. We can more easily change it, if needed (e.g. more checkboxes to be added).